### PR TITLE
Update Envisalink component configuration

### PR DESCRIPTION
**Pull request by klaasnicolaas**
**Description:**
Update style of Envisalink component documentation to follow new configuration variables description.
Related to #6385.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Please help, do not see where the problem may be.

File: source/_components/envisalink.markdown [hunk 1]
```diff
@@ -54,20 +54,76 @@ envisalink:
...
+  required: false
+  default: 60
+  type: integer
+zonedump_interval:
+  description: This is an interval (measured in seconds) where the evl will dump out all zone statuses. This is required for Honeywell systems, which do not properly send zone closure events. DSC boards do not technically need this.
+  required: false
+  default: 30
+  type: integer
+panic_type:
+  description: "Both DSC and Honeywell boards support a "panic" alarm. This is used when the alarm_trigger service is called in Home Assistant. This determines which type of panic alarm to raise. Valid values are: Police, Fire, Ambulance."
```
Comment (DubhAd) [inline source/_components/envisalink.markdown]: Remove the `:` after `are` (and dump the quotes here and elsewhere)

**Review by DubhAd — CHANGES_REQUESTED** [review]

**Review by DubhAd — APPROVED** [review]

File: source/_components/envisalink.markdown [hunk 2]
```diff
@@ -54,20 +54,76 @@ envisalink:
...
+  required: false
+  default: 3
+  type: integer
+keepalive_interval:
+  description: This is a periodic heartbeat signal (measured in seconds) sent to your Envisalink board to keep it from restarting. This is required for DSC and Honeywell systems.
+  required: false
+  default: 60
+  type: integer
+zonedump_interval:
+  description: This is an interval (measured in seconds) where the evl will dump out all zone statuses. This is required for Honeywell systems, which do not properly send zone closure events. DSC boards do not technically need this.
```
Comment (DubhAd) [inline source/_components/envisalink.markdown]: Please exapand *evl*

**Review by DubhAd — CHANGES_REQUESTED** [review]

**Review by DubhAd — APPROVED** [review]

**Review by fabaff — APPROVED** [review]
Thanks :bird:

**Pull request merged**